blackjack/blackjack_test.go

```diff
@@ -33,23 +34,37 @@ func TestNewBlackjackGame(t *testing.T) {
 		Cards: stack,
 	}
 
+	setupOutput := &bytes.Buffer{}
+	setupInput := strings.NewReader("1\nPlanty\na\n3")
+
 	g, err := blackjack.NewBlackjackGame(
 		blackjack.WithCustomDeck(deck),
+		blackjack.WithOutput(setupOutput),
+		blackjack.WithInput(setupInput),
 	)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	g.Start()
-
-	got := blackjack.ReportMap[g.Player.HandOutcome]
+	blackjack.RunCLI(g)
 
-	want := "***** Blackjack!  Player wins *****"
+	want := 3
+	got := g.Players[0].Win + g.Players[0].Lose + g.Players[0].Tie
```

Comment:
I think I see what this is doing, but it's a little obscure. We could imagine an incorrect game outcome that would produce the same sum, can't we? If what we need to establish is that the players wins 1 time, loses 2, and ties 0, we should test exactly that.

Reply:
(It would be convenient for the test if this information were in the form of a struct, so if it's convenient for the test, it's probably convenient for users too, isn't it?)